README.rst

```diff
@@ -85,6 +85,7 @@ Example
 
    static void Main(string[] args)
    {
+       Runtime.PythonDLL = "python38.dll";
```

Comment:
I really don't want to give people the idea that this is the way to go. Maybe we can point them toward (or include) https://github.com/losttech/WhichPython? Maybe give a note above to set the `PYTHONNET_PYDLL` environment variable?

Reply:
Yeah, I wasn't really keen on this either, but I figured since this was one of the examples given in the paragraph above it that this would be at least acceptable.
I also thought about mentioning the env var, but I didn't want to clutter this example with a comment (at least not without the permisson of you guys)